environments/common/application-load-balancer/locals.tf

```diff
@@ -0,0 +1,44 @@
+locals {
+  services = {
+    admin = {
+      target_group_name = "trade-tariff-ad-tg-${var.environment}"
```

Comment:
Do we need the prefix `trade-tariff` if we're restricted to 30 chars?

Reply:
Also, probably we don't need `tg` since this is clear from the variable/resource you're creating?

Reply:
`"admin-${var.environment}"`

Reply:
Yeah it's a good point. I've just left it there because recreating them is slightly awkward; we'd then have to go into each service we've done the Terraform for and change the data source for the target group. But ideally they should just be the name of the service.

I _can_ change it and just **Sort It Out**:tm: by issuing a few PRs and resolving it in one go, though.